frontend/asfui/app/management/commands/remaster_output.py

```diff
@@ -193,6 +196,80 @@ def parser_nuclei_http(PARSER_INPUT, PARSER_OUTPUT):
             debug("Found nothing:"+line)
     return
 
+
+def parser_wpscan_json(PARSER_INPUT, PARSER_OUTPUT):
+    print("Sending to parser for WPScan")
+    #ParseWPscanData(PARSER_INPUT)
+    username = urllib.parse.quote_plus(settings.MONGO_USER)
+    password = urllib.parse.quote_plus(settings.MONGO_PASSWORD)
+    url = settings.MONGO_URL
+    port = settings.MONGO_PORT
+    myclient = pymongo.MongoClient(f"mongodb://{username}:{password}@{url}:{port}")
+    db = myclient["Nuclei"]
+    Collection = db["report"]
+    file_data = json.load(PARSER_INPUT)
+    primaryBulkArr = []
+    result_list =[]
+    severity = 'medium'
+    try:
+        host = file_data['target_url']
+        start_time = file_data['start_time']
+        timestamp = (datetime.fromtimestamp(start_time)).strftime("%Y-%m-%dT%H:%M:%S")
+        if "vulnerabilities" in file_data['version']:
+            for i in file_data['version']['vulnerabilities']:
+                #print(i["title"], data["target_url"])   
+                row = {'host': host, 'jira_created': False, 'verified':False, "ignored":False, 'info':{'name':i['title'], 'description': i['title'], 'severity': severity}, 'matched-at':file_data['target_url'], 'template-id': i['title'], 'timestamp':timestamp }
+                result_list.append(row)
+        if "vulnerabilities" in file_data["main_theme"]:
+            for i in file_data["main_theme"]["vulnerabilities"]:
+                row = {'host': host, 'jira_created': False, 'verified':False, "ignored":False, 'info':{'name':i['title'], 'description': i['title'], 'severity': severity}, 'matched-at':file_data['main_theme']['location'], 'template-id': i['title'], 'timestamp':timestamp }
+                result_list.append(row) 
+        for i in file_data["plugins"]:
+            if "vulnerabilities" in file_data["plugins"][i]:
+                    for j in file_data["plugins"][i]["vulnerabilities"]:
+                        if "title" in j:
+                            row = {'host': host, 'jira_created': False, 'verified':False, "ignored":False, 'info':{'name':j['title'], 'description': j['title'], 'severity': severity}, 'matched-at':file_data["plugins"][i]['location'], 'template-id': j['title'], 'timestamp':timestamp }
+                            result_list.append(row) 
+    except Exception as e:
+        print("error in parsing WPscan report:", str(e))
+        pass
+        
+    for x in result_list:
+        primaryBulkArr.append(pymongo.UpdateOne({"template-id": x['template-id'], 'host':x['host'], 'matched-at':x['matched-at']}, {'$set':x, '$inc':{'counter':1}}, upsert=True))
+    if len(primaryBulkArr)> 0:
+        Collection.bulk_write(primaryBulkArr)
+    else:
+        print("nothing to insert")
+    
+    return 
+    
+
+
+def parser_nuclei_json(PARSER_INPUT, PARSER_OUTPUT):
+    username = urllib.parse.quote_plus(settings.MONGO_USER)
+    password = urllib.parse.quote_plus(settings.MONGO_PASSWORD)
+    url = settings.MONGO_URL
```

Comment:
Same comment as suggested in https://github.com/vmware-labs/attack-surface-framework/pull/18/files#diff-ccb2b112e818f612867f84573df1c02da2d80a2832dcda026b492da103e85078R45